### PR TITLE
Add DolphinScheduler client tool

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _data/tools.yml [hunk 1]
```diff
@@ -134,6 +134,25 @@
...
+    Apache DolphinScheduler is an open-source, distributed workflow scheduling
+    platform designed to manage and execute batch jobs, data pipelines, and ETL
+    processes. DolphinScheduler enables users to create and manage consecutive
+    jobs run easily, including support for different types of tasks, such as SQL
+    statements, shell scripts, Spark jobs, Kubernetes deployments, and many
+    others. 
+  links:
+    - urltext: Apache DolphinScheduler
+      url: https://dolphinscheduler.apache.org/
+    - urltext: Apache DolphinScheduler documentation
```
Comment (mosabua) [inline _data/tools.yml:152]: Lets leave the documentation link out for now. We should ping them to add a Trino specific page..

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor change

**Comment by mosabua** [comment]
Also .. you need to rebase..

**Comment by ryandoowan** [comment]
Ok something happened when I tried to rebase, I'm making a new PR

**Pull request closed without merging**